recipes/libwebp/all/conanfile.py

```diff
@@ -85,5 +85,7 @@ def package_info(self):
         self.cpp_info.libs = ['webpmux', 'webpdemux', 'webpdecoder', 'webp']
         if self.options.shared and self.settings.os == "Windows" and self.settings.compiler != 'Visual Studio':
             self.cpp_info.libs = [lib + '.dll' for lib in self.cpp_info.libs]
-        if self.settings.os == "Linux":
-            self.cpp_info.libs.append("pthread")
+        if self.settings.os != "Windows":
```

Comment:
Same comment, limit changes to Android. It might have collateral issues for NIX systems. 
Please do different PRs to change different libraries.
Thanks!